jobs/build/ocp3/Jenkinsfile

```diff
@@ -684,6 +684,12 @@ node {
                             --filter-by-os amd64
                             --push-to-defaults ${ODCS_OPT}
                         """
+                        buildlib.doozer """
```

Comment:
@joepvd Could you add a comment about the hack so we won't forget the reason to do this?
Otherwise LGTM